content/docs/deployment/csmoperator/drivers/powerflex.md

```diff
@@ -105,38 +105,38 @@ For detailed PowerFlex installation procedure, see the [Dell PowerFlex Deploymen
       password: "Password123"
       systemID: "2b11bb111111bb1b"
       endpoint: "https://127.0.0.2"
-      skipCertificateValidation: true 
+      skipCertificateValidation: true
       mdm: "10.0.0.3,10.0.0.4"
     ```
 
     If replication feature is enabled, ensure the secret includes all the PowerFlex arrays involved in replication.
 
     After editing the file, run this command to create a secret called `vxflexos-config`.
     ```bash
-    
+
     kubectl create secret generic vxflexos-config -n vxflexos --from-file=config=secret.yaml
     ```
 
     Use this command to replace or update the secret:
 
     ```bash
-    
+
     kubectl create secret generic vxflexos-config -n vxflexos --from-file=config=secret.yaml -o yaml --dry-run=client | kubectl replace -f -
     ```
 
 ### Install Driver
 
 1. Follow all the [prerequisites](#prerequisite) above
-   
-2. Create a CR (Custom Resource) for PowerFlex using the sample files provided 
+
+2. Create a CR (Custom Resource) for PowerFlex using the sample files provided
    [here](https://github.com/dell/csm-operator/tree/master/samples). This file can be modified to use custom parameters if needed.
 
 3. Users should configure the parameters in CR. The following table lists the primary configurable parameters of the PowerFlex driver and their default values:
 
    | Parameter | Description | Required | Default |
    | --------- | ----------- | -------- |-------- |
    | dnsPolicy | Determines the DNS Policy of the Node service | Yes | ClusterFirstWithHostNet |
-   | fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType` | No | "ReadWriteOnceWithFSType" |
+   | fsGroupPolicy | Defines which FS Group policy mode to be used, Supported modes `None, File and ReadWriteOnceWithFSType`. In OCP <= 4.16 and K8s <= 1.29, fsGroupPolicy is an immutable field. | No | "ReadWriteOnceWithFSType" |
```

Comment:
Hi @shaynafinocchiaro 
I guess we need similar note for Unity driver. Please check.

Reply:
Fixed